src/features/CareViewPatientsSummary/components/SlotDetailsCell.jsx

```diff
@@ -126,7 +126,11 @@ export const SlotDetailsCell = ({
             </div>
             <span>{epochTo24HourTimeFormat(slotItem.startTime)}</span>
             <div className="drug-details-wrapper">
-              <span>{slotItem.order.drug.display}</span>
+              {drugNonCoded !== null ? (
+                <span>{drugNonCoded}</span>
+              ) : (
+                <span>{slotItem?.order?.drug?.display}</span>
+              )}
```

Comment:
we can write like this "**<_span_>**{drugNonCoded ? drugNonCoded : slotItem?.order?.drug?.display}<**/span**>"